src/objects/RecurringDonationChangeHistory__c.object

```diff
@@ -0,0 +1,389 @@
+        <label>Previous Schedule</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Previous_Status_Reason__c</fullName>
+        <description>The previous Status Reason for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The previous Status Reason for the related Recurring Donation.</inlineHelpText>
+        <label>Previous Status Reason</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Previous_Status__c</fullName>
+        <description>The previous Status for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The previous Status for the related Recurring Donation.</inlineHelpText>
+        <label>Previous Status</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Previous_Yearly_Value__c</fullName>
+        <description>The previous yearly value for this Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The previous yearly value for this Recurring Donation.</inlineHelpText>
+        <label>Previous Yearly Value</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>2</scale>
+        <trackTrending>false</trackTrending>
+        <type>Currency</type>
+    </fields>
+    <fields>
+        <fullName>Recurring_Donation__c</fullName>
+        <description>The Recurring Donation related to this Change History record.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Recurring Donation related to this Change History record.</inlineHelpText>
+        <label>Recurring Donation</label>
+        <referenceTo>npe03__Recurring_Donation__c</referenceTo>
+        <relationshipLabel>Recurring Donation Change History</relationshipLabel>
+        <relationshipName>RD_Change_History</relationshipName>
+        <relationshipOrder>0</relationshipOrder>
+        <reparentableMasterDetail>false</reparentableMasterDetail>
+        <trackTrending>false</trackTrending>
+        <type>MasterDetail</type>
+        <writeRequiresMasterRead>false</writeRequiresMasterRead>
+    </fields>
+    <label>Recurring Donation Change History</label>
+    <nameField>
+        <displayFormat>RDCH-{0000000}</displayFormat>
+        <label>Recurring Donation Change History ID</label>
+        <type>AutoNumber</type>
+    </nameField>
+    <pluralLabel>Recurring Donation Change History</pluralLabel>
```

Comment:
```suggestion
    <pluralLabel>Recurring Donation Change Histories</pluralLabel>
```

Reply:
Ha, I had that originally, but thought it looked funny in the Related List 🤷‍♂️

Reply:
You know, I think you're right actually. The standard field history tracking uses the singular version too. Please revert my suggestion! @lparrott

Reply:
@lparrott Just wanted to check on a few of the fields on the History object: 

- New Payment Method / Previous Payment Method fields
- New Schedule / Previous Schedule fields
- New Status / Previous Status fields
- New Status Reason / Previous Status Reason fields

The equivalent fields on the RD object are picklist fields and the values in the picklists and NPSP ships translations of the picklist values for the the languages we support. How are these text fields going to work for users of the languages we support, or multinational orgs that may have users in more than one language using Recurring Donations? Or are we saying the values in these history fields will be English only?

Reply:
@jamesbsfdc The field values in RD Change History can either be the picklist selection's Label, which will be translated to the current User's language (I believe), or the picklist selection's API name, which will be uniform for all Users, but will most likely be in English (at least, for the options we provide by default). We haven't implemented this yet, does one option sound better than the other?

Having the History fields be **matching** picklists would be ideal, but then we run into issues with keeping them in sync and up to date with each other.

Reply:
Thanks for the detail @lparrott! Sounds like both come with tradeoffs for international audiences. 

I guess using the user's language rather than API values sounds like the least bad option if they have to be text values. This would work for the customers whose org is using one language across the board for all users (be this English or any other language). 

The issue is for customers that may have English and Spanish users. If they have users with visibility to all change history data they will see English and Spanish values across the data, which may also make reporting and challenging.  I thinking of international nonprofits here who have gone for a one-org approach rather than an org by country approach. They may have shared resource teams (e.g a data analysis team) who review data across the business (country) units.

Interested to hear what others think. Think this may be important to document @samknox  @mpusto ?